api/v1alpha1/database_types.go

```diff
@@ -98,11 +101,10 @@ type Instance struct {
 	TimeZone string `json:"timezone"`
 	Type     string `json:"type"`
 	// +optional
-	Profiles Profiles `json:"profiles"`
+	Profiles *Profiles `json:"profiles"`
```

Comment:
Why are these changes showing up in this PR? These don't seem to be related to this PR...